src/Orchard.Cms.Web/Modules/Orchard.OpenId/Recipes/OpenIdSettingsStep.cs

```diff
@@ -26,28 +28,27 @@ public override string Name
             get { return "OpenIdSettings"; }
         }
 
-        public override async Task ExecuteAsync(RecipeExecutionContext recipeContext)
+        public override async Task ExecuteAsync(RecipeExecutionContext context)
         {
-            var model = recipeContext.RecipeStep.Step.ToObject<OpenIdSettingsStepModel>();
+            var model = context.RecipeStep.Step.ToObject<OpenIdSettingsStepModel>();
 
-            var openIdSettings = await _openIdService.GetOpenIdSettingsAsync();
-            openIdSettings.TestingModeEnabled = model.TestingModeEnabled;
-            openIdSettings.DefaultTokenFormat = model.DefaultTokenFormat;
-            openIdSettings.Audiences = model.Audiences;
-            openIdSettings.Authority = model.Authority;
-            openIdSettings.CertificateStoreLocation = model.CertificateStoreLocation;
-            openIdSettings.CertificateStoreName = model.CertificateStoreName;
-            openIdSettings.CertificateThumbPrint = model.CertificateThumbPrint;
-            
-            await _openIdService.UpdateOpenIdSettingsAsync(openIdSettings);
+            var settings = await _openIdService.GetOpenIdSettingsAsync();
+            settings.TestingModeEnabled = model.TestingModeEnabled;
+            settings.AccessTokenFormat = model.AccessTokenFormat;
+            settings.Audiences = model.Audiences;
+            settings.Authority = model.Authority;
+            settings.CertificateStoreLocation = model.CertificateStoreLocation;
+            settings.CertificateStoreName = model.CertificateStoreName;
+            settings.CertificateThumbPrint = model.CertificateThumbPrint;
 
+            await _openIdService.UpdateOpenIdSettingsAsync(settings);
         }
     }
 
     public class OpenIdSettingsStepModel
     {
         public bool TestingModeEnabled { get; set; } = false;
-        public TokenFormat DefaultTokenFormat { get; set; } = TokenFormat.JWT;
+        public TokenFormat AccessTokenFormat { get; set; } = TokenFormat.Encrypted;
```

Comment:
https://github.com/OrchardCMS/Orchard2/issues/310

Main reason: it's "more secure".